### ISSUE TITLE
Issue Reproducing Results – Missing Dynamic Teacher JSON File

### ISSUE THREAD

**Issue by Dacaidi** [untitledunmastered1998/CIL-LWN#1]
Hi,

Thank you for sharing your work and the accompanying code. I really appreciate it!

I was wondering if you could provide the JSON file for the dynamic teacher settings? I believe the file exps/online_dynamic.json is intended for this, but I noticed that the hyperparameters in the file don’t seem to fully match those mentioned in the paper. Additionally, some important hyperparameters for the dynamic teacher, such as the number of epochs for the initial task, appear to be missing.

It would be extremely helpful if you could share the correct JSON file so that I can accurately reproduce your results.

Thank you in advance for your help!

**Comment by Dacaidi** [comment]
Hi,

I attempted to reproduce the dynamic teacher experiment, but some training parameters (e.g., initial and incremental epochs) seem to be missing from the code. I used the parameters provided in your paper, but my CIFAR100 (10/10) results show an average accuracy of 19.76%, compared to the 38.42% reported. I've attached the log for reference.
[CIL_1993_mobilenet_v2.log](https://github.com/user-attachments/files/17457159/CIL_1993_mobilenet_v2.log)


Could you kindly share the dynamic teacher's JSON file and the LwF training parameters and hyperparameters? That would be greatly appreciated!

Thanks again!
